examples/multi-account/main.tf

```diff
@@ -0,0 +1,76 @@
+locals {
+  namespace   = "namespace"
+  environment = "environment"
+  stage       = "stage"
+  subscriptions = {
+    "s-n-attribute1" = {
+      enabled    = true
+      stage      = "stage"
+      name       = "name"
+      attributes = ["attribute1"]
+    }
+    "s-n-attribute2" = {
+      enabled    = true
+      stage      = "stage"
+      name       = "name"
+      attributes = ["attribute2"]
+    }
+  }
+}
+
+module "topic" {
+  source   = "github.com/justtrackio/terraform-aws-sns-topic?ref=v1.0.0"
+  for_each = local.subscriptions
+
+  providers = {
+    aws = aws.topic_owner
+  }
+
+  principals_with_subscribe_permission = [
+    "arn:aws:iam::123456789123:root"
+  ]
+
+  alarm_create = false
+  namespace    = local.namespace
+  environment  = local.environment
+  stage        = each.value.stage
+  name         = each.value.name
+  attributes   = each.value.attributes
+}
+
+module "queue" {
+  source = "../.."
```

Comment:
usually on github hosted terraform modules you use the ../../ version to always reflect the version you currently have checked out. That should also garantee that upon action run your tf-recommit hook would fail if your examples syntax doesn't work anymore. Also would not make sense in a way that you as a contributor can not be sure about which version your changeset may be included in.